perl/README.md

```diff
@@ -0,0 +1,1025 @@
+
+For exampole, Bystro can afford to provide complete annotations from gnomad v4.1, for all gnomAD populations and subpopulations, from the exomes, genomes, and joint datasets, genome-wide. This means intersecting terabytes of data over the entire genome with each individual variant, all done in microseconds per variant. **No other tool can do this**.
+
+## Running Your First Annotation
+
+See the [INSTALL.md#configuring-the-bystro-annotator](./INSTALL.md#configuring-the-bystro-annotator) section for instructions on how to configure the Bystro Annotator
+
+```sh
+bystro-annotate.pl --config ~/bystro/config/hg38.yml --threads 32 --input gnomad.genomes.v4.0.sites.chr22.vcf.bgz --output test/my_annotation --compress gz
+```
+
+The above command will annotate the `gnomad.genomes.v4.0.sites.chr22.vcf.bgz` file with the hg38 database, using 32 threads, and output the results to `test`, and will use `my_annotation` as the prefix for output files.
+
+The result of this command will be:
+
+```sh
+Created completion file
+{
+   "error" : null,
+   "totalProgress" : 8599234,
+   "totalSkipped" : 0,
+   "results" : {
+      "header" : "my_annotation.annotation.header.json",
+      "sampleList" : "my_annotation.sample_list",
+      "annotation" : "my_annotation.annotation.tsv.gz",
+      "dosageMatrixOutPath" : "my_annotation.dosage.feather",
+      "config" : "hg38.yml",
+      "log" : "my_annotation.annotation.log.txt",
+      "statistics" : {
+         "qc" : "my_annotation.statistics.qc.tsv",
+         "json" : "my_annotation.statistics.json",
+         "tab" : "my_annotation.statistics.tsv"
+      }
+   }
+}
+```
+
+Explanation of the output:
+
+- `my_annotation.annotation.header.json`: The header of the annotated dataset
+
+- `my_annotation.sample_list`: The list of samples in the annotated dataset
+
+- `my_annotation.annotation.tsv.gz`: A block gzipped TSV file with one row per variant and one column per annotation. Can be decompressed with `bgzip` or any program compatible with the gzip format, like `gzip` and `pigz`.
+
+- `my_annotation.dosage.feather`: The dosage matrix file, where the first column is the `locus` column in the format "chr:pos:ref:alt", and columns following that are sample columns, with the dosage of the variant for that sample (0 for homozygous reference, 1 for 1 copy of the alternate allele, 2 for 2, and so on). -1 indicates missing genotypes. The dosage is the expected number of alternate alleles, given the genotype. This is useful for downstream analyses like imputation, or for calculating polygenic risk scores
```

Comment:
How about for extra copies, are they still 2? I don't remember, are trisomic sites generally handled?

Reply:
we talk about that

with the dosage of the variant for that sample (0 for homozygous reference, 1 for 1 copy of the alternate allele, 2 for 2, and so on)

Reply:
I'll add a note that we do not assume diploid